receiver/gitproviderreceiver/metadata.yaml

```diff
@@ -41,48 +41,83 @@ attributes:
 metrics:
   git.repository.count:
     enabled: true
-    description: Number of repositories in an organization
-    unit: "{repository}"
+    description: The number of repositories in an organization.
+    unit: '{repository}'
     gauge:
       value_type: int
     attributes: []
   git.repository.branch.count:
     enabled: true
-    description: Number of branches in a repository
-    unit: "{branch}"
+    description: The number of branches in a repository.
+    unit: '{branch}'
     gauge:
       value_type: int
     attributes: [repository.name]
+  git.repository.branch.time:
```

Comment:
Naming this `git.repository.branch.time_open` would make it more in line with `git.repository.pull_request.time_open`, but I'm not sure if it's better. What do you think?

Reply:
> time - an instrument that measures passage of time should be called entity.time. For example, system.cpu.time with attribute state = idle | user | system | .... time measurements are not necessarily wall time and can be less than or greater than the real wall time between measurements.
> 
> time instruments are a special case of usage metrics, where the limit can usually be calculated as the sum of time over all attribute values. utilization for time instruments can be derived automatically using metric event timestamps. For example, system.cpu.utilization is defined as the difference in system.cpu.time measurements divided by the elapsed time and number of CPUs.

That's from the metric doc on the convention. That's why I went with `time` originally. If we feel `time_open` is more clear & inline with the convention then I'm good to change this to `time_open`. 

Btw, every time I change these I go in and change the underlying dashboards that are published on the grafana dashboard market to support these new metric names. And I plan on going through and changing this all again to match the `scm...` attribute convention once that gets merged.